Makefile

```diff
@@ -89,17 +89,6 @@ check-pip-requirements:
 		batch \
 		ci
 
-.PHONY: check-linux-pip-requirements
-check-linux-pip-requirements:
-	./check_linux_pip_requirements.sh \
```

Comment:
This target is unused and the `check_linux_pip_requirements.sh` file doesn't exist.